src/components/topic/Description.js

```diff
@@ -81,21 +114,168 @@ const TeamDescriptiton = styled.div`
       color: ${colors.primaryColor};
     }
   }
+  /* Since the component corresponding to unstyled type uses a className to address text-align as justify.
+   * Boost the specificity of the override style by && instead of & here.
+   */
+  && * {
+    font-size: ${typography.font.size.small};
+    color: ${colors.gray.gray50};
+    line-height: ${lineHeight.lineHeightLarge};
+    text-align: center;
+    margin-bottom: 0;
+  }
+`
+
+const BlockQuote = styled.blockquote`
+  font-style: italic;
+  padding-left: 1rem;
+  line-height: 1.85;
+  font-size: 1rem;
+  font-weight: 300;
+  white-space: pre-wrap;
+  margin: 0 1.33rem 0 1.33rem;
+`
+
+const YoutubeContainer = styled.div`
+  width: 100%;
+  &&& * {
+    margin-bottom: 0;
+  }
 `
 
+const YoutubeIframeContainer = styled.div`
+  max-width: 100%;
+  margin-bottom: 0;
+  iframe,object,embed {
+    width: 560px;
+    height: 315px;
+  }
+`
+
+const DescTextBlock = styled.div`
+  color: #808080;
+  font-size: 0.88rem;
+  line-height: 1.8;
+  margin: 0.94rem 1.33rem 0 1.33rem;
+}
+`
+
+const TopicDescriptionJSX = {
+  BlockQuote: (content) => {
+    return (
+      <BlockQuote dangerouslySetInnerHTML={{ __html: _.get(content, [ 0 ], '') }}></BlockQuote>
+    )
+  },
+  Youtube: (content) => {
+    let { description, youtubeId } = _.get(content, [ 0 ], {})
+    if (!youtubeId) {
+      return null
+    }
+    return (
+      <YoutubeContainer>
+        <YoutubeIframeContainer>
+          <YoutubePlayer videoId={youtubeId} />
+        </YoutubeIframeContainer>
+        <DescTextBlock>
+          {description}
+        </DescTextBlock>
+      </YoutubeContainer>
+    )
+  }
+}
+
+const TopicTeamJSX = {
+  Image: (content) => {
+    const imgSrc = _.get(content, [ 0, 'tablet', 'url' ], '')
+    return (
+      <img src={imgSrc} />
+    )
+  }
+}
+
+const getTopicContent = (data, contentType) => {
+  if (Array.isArray(data)) {
+    return data.map((ele) => {
+      let styles = {}
+      let type = ele.type
+      let ArticleComponent = null
+      let TopicComponent = null
+      if (contentType === 'description') {
+        switch(type) {
+          case 'blockquote':
+            TopicComponent = TopicDescriptionJSX.BlockQuote
+            break
+          case 'youtube':
+            TopicComponent = TopicDescriptionJSX.Youtube
+            break
+          default:
+            ArticleComponent = getArticleComponent(type)
+        }
+      } else if (contentType === 'team') {
+        switch(type) {
+          case 'image':
+            TopicComponent = TopicTeamJSX.Image
+            break
+          default:
+            ArticleComponent = getArticleComponent(type)
+        }
+      }
+
+      if (!ArticleComponent && !TopicComponent) {
+        return null
+      }
```

Comment:
Do we need  both `ArticleComponent` and `TopicComponent` here?
Could we only use `Component` instead?